crankshaft-docker/src/events.rs

```diff
@@ -0,0 +1,32 @@
+//! Event utils
+use std::time::SystemTime;
+use std::time::UNIX_EPOCH;
+
+use crankshaft_monitor::proto::Event;
+use crankshaft_monitor::proto::EventType;
+use tokio::sync::broadcast;
+
+/// Sends a Event through a braodcast channel
+pub fn send_event(
```

Comment:
One problem with the implementation of this method is that in many places we're calling it with a `format!` for the message, which is allocating the message even when `sender` is `None`.

Perhaps we can change this to a macro which will effectively inline below so that the `message` expression is only run when `sender` is `Some`?